integrations-and-supported-tools/pytorch/notebooks/Neptune_PyTorch.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Neptune + PyTorch\n",
+    "\n",
+    "Introduction\n",
+    "\n",
+    "This guide will show you how to:\n",
+    "\n",
+    "* Create a NeptuneLogger()\n",
+    "* Log training metrics to Neptune using NeptuneLogger()\n",
+    "* Upload model checkpoints to Neptune using NeptuneLogger()\n",
+    "* Log model predictions to Neptune using NeptuneLogger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FUaDrmRu-h3k"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "* If you're running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example logged to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_0KXqlePqNi"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i36OvZAgvxkj"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "! pip install neptune[pytorch] torch torchvision numpy torchviz\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rik2HDr0r5Hn",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "## Start a run\n",
+    "\n",
+    "To create a new run for tracking the metadata, we tell Neptune:\n",
+    "* **Who you are** - with your Neptune API token\n",
+    "* **Where to send the metadata** - to your Neptune project\n",
+    "\n",
+    "For example, if your workspace name is `ml-team` and the project name is `classification`, the project argument is: `project=\"ml-team/classification\"`.\n",
+    "\n",
+    "To find your API token and project name, [log in to Neptune](https://app.neptune.ai/).\n",
+    "- In the top-right corner, click your avatar and select **Get your API token**.\n",
+    "- To find and copy your project name, navigate to the project, then click **Settings** → **Properties**.\n",
+    "\n",
+    "### Haven't registered yet?\n",
+    "\n",
+    "To log anonymously to a public project, replace the code below with the following:\n",
+    "\n",
+    "```python\n",
+    "import neptune\n",
+    "\n",
+    "run = neptune.init_run(project=\"common/pytorch-integration\", api_token=neptune.ANONYMOUS_API_TOKEN)\n",
+    "```"
```

Comment:
We can leave this section out, as the code cell has the anonymous/public logging by default.